queue/message_prioritizer.go

```diff
@@ -0,0 +1,185 @@
+package queue
+
+import (
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/pkg/errors"
+)
+
+type State struct {
+	HasRunningInstance bool
+	Height             qbft.Height
+	Slot               phase0.Slot
+	Quorum             uint64
+}
+
+// DecodedSSVMessage is a types.SSVMessage accompanied by it's decoding.
+type DecodedSSVMessage struct {
+	*types.SSVMessage
+
+	// Body is the decoded Data.
+	Body interface{} // *SignedMessage | *SignedPartialSignatureMessage
+}
+
+func DecodeSSVMessage(m *types.SSVMessage) (*DecodedSSVMessage, error) {
```

Comment:
We can utilize this in other places, extract `DecodedSSVMessage` and place it with `types.SSVMessage`

Reply:
Good idea

Reply:
We moved the `queue` package under `ssv`.